### DISCUSSION TITLE
Items in "Shipped to Customer"

### DISCUSSION THREAD

**Discussion by Pixeltrace-GER** [inventree/InvenTree#3708] [Q&A] [answered]
Hi there,
I have a short understanding question.

If I create a sales order, allocate stock and marked the part as shipped. It moved to category "Shipped to Customer". What happens with this entry? Will it stay there forever?

Thanks in advance

**Comment by matmair** [comment, marked as answer]
Hi there @Pixeltrace-GER !
Yes, the entries normally stay there indefinetly.

Regards, Matthias

**Reply by Pixeltrace-GER** [reply]
Matthias, thank you for the quick reply. 
So, I haven't missed a step in the workflow.

**Comment by winzkigermany** [comment]
Could we please have a setting to delete such shipped stock?
(like in the same way that depleted stock is deleted)
At the moment, I have to exclude such stock by filtering "Sent to customer=false", but that's cumbersome.
And the amount of shipped stock is already disturbingly high after one week of use.

**Reply by miggland** [reply]
Just a note: the filter "Available=true" should have the same effect..

**Reply by matmair** [reply]
Please use feature requests for feature requests and not old discussions. We can not track the state of discussions well andmight not even get notified about changes in old ones.
